src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -289,12 +298,13 @@ class ActivityFeed extends React.Component<Props, State> {
 
         return (
             // eslint-disable-next-line
-            <div className="bcs-activity-feed" data-testid="activityfeed" onKeyDown={this.onKeyDown}>
+            <div className={classNames("bcs-activity-feed", { "bcs-is-scrolled": this.state.isScrolled })} data-testid="activityfeed" onKeyDown={this.onKeyDown}>
                 <div
                     ref={ref => {
                         this.feedContainer = ref;
                     }}
                     className="bcs-activity-feed-items-container"
+                    onScroll={this.handleToggleHeaderShadow}
```

Comment:
We throttle this event handler in most places, since it runs very often. Should we do so here, as well?